zlib.sh

```
@@ -3,7 +3,8 @@ version: "%(tag_basename)s"
 source: https://github.com/star-externals/zlib
 tag: v1.2.8
 build_requires:
- - "GCC-Toolchain:(?!osx)"
+ - GCC-Toolchain
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)